### PR TITLE
code tidy & externalizing some config

### PR DESCRIPTION
**Pull request by matallen**

### CODE REVIEW

**Comment by etsauer** [comment]
@matallen is there a dummy config we can use to test?

**Comment by matallen** [comment]
> @matallen is there a dummy config we can use to test?

Yes, the environment settings I use are here: https://gitlab.consulting.redhat.com/mallen/ninja-board

although watch out for the graphs proxy url as that is the LIVE version, just change the url a bit as we dont have a dev instance of the graphs proxy deployed.

File: src/main/webapp/mojo-miniBeltWidget,jsp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    document.getElementById("_name").innerHTML=json.displayName;
+    document.getElementById("_level").src=ctx+"/images/"+json.level.toLowerCase()+"_belt_icon.png";
+  }
+
+  function getUsername(){
+    if(undefined!=window.parent._jive_current_user){
+      var username=window.parent._jive_current_user.username;
+      var displayName=window.parent._jive_current_user.displayName;
+    }
+    //if(username==undefined) username="mallen";
```
Comment (sabre1041) [inline src/main/webapp/mojo-miniBeltWidget,jsp]: Can this be removed?

Comment (matallen) [reply src/main/webapp/mojo-miniBeltWidget,jsp]: Yes, but I'll do it on a subsequent commit - I had to wipe my fork so I'm unsure how I'd make that change now.

**Pull request merged**